src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.VisualStudio.Language.Intellisense;
+using GitHub.Extensions;
+using GitHub.InlineReviews.ViewModels;
+
+namespace GitHub.InlineReviews.Peek
+{
+    class InlineCommentPeekResult : IPeekResult
+    {
+        public InlineCommentPeekResult(InlineCommentPeekViewModel viewModel)
+        {
+            Guard.ArgumentNotNull(viewModel, nameof(viewModel));
+
+            this.ViewModel = viewModel;
+        }
+
+        public bool CanNavigateTo => true;
+        public InlineCommentPeekViewModel ViewModel { get; }
+
+        public IPeekResultDisplayInfo DisplayInfo { get; }
+            = new PeekResultDisplayInfo("Review", null, "GitHub Review", "GitHub Review");
+
+        public Action<IPeekResult, object, object> PostNavigationCallback => null;
+
+        public event EventHandler Disposed;
+
+        public void Dispose()
```

Comment:
I don't think there's actually anything we need to dispose here. The reason this method appears is that `IPeekResult` implements `IDisposable` so we need to implement it too.